README.md

```diff
@@ -57,3 +57,92 @@ Releases are automatically pushed on commits merged to master that fulfill the f
 The release process is building all runtimes and then puts them into a release in this github repository.
 
 The format of [`CHANGELOG.md`](CHANGELOG.md) is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+# Weight Generation
+
+To generate weights for a runtime
+
+1. Build `chain-spec-generator` with `--profile production --features runtime-benchmarks`
+2. Use it to build a chain spec for your runtime, e.g. `./target/production/chain-spec-generator --raw polkadot-local > polkadot-chain-spec.json`
+3. Create `file_header.txt`
+
+```text
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+```
+
+4. `rsync` chain spec/s and the file header to a benchmark machine
+
+5. Build `polkadot-sdk` with `--profile production --features runtime-benchmarks` on the benchmark machine
```

Comment:
```suggestion
5. Build `polkadot` binary from the latest release of `polkadot-sdk` with `--profile production --features runtime-benchmarks --bin polkadot` on the benchmark machine
```

I would specify here "latest release version" as Basti suggested [here](https://github.com/polkadot-fellows/runtimes/issues/50#issuecomment-1738925780). And also that just `--bin polkadot` is needed (not to waste resources by building the whole `polkadot-sdk` repo)